packages/test/test-end-to-end-tests/src/test/gc/gcTombstone.spec.ts

```diff
@@ -309,4 +309,93 @@ describeNoCompat("GC DataStore Tombstoned When It Is Sweep Ready", (getTestObjec
         assert(closeError.errorType === "dataCorruptionError");
         assert(closeError.message === "Context is tombstoned: Call site -  process!");
     });
+
+    // If this test starts failing due to runtime is closed errors try first adjusting `sweepTimeoutMs` above
+    itExpects("Container loaded after sweep timeout expires is unable to load tombstoned datastores",
+    [
+        {
+            eventName: "fluid:telemetry:Summarizer:Running:SweepReadyObject_Loaded",
+        },
+    ],
+    async () => {
+        const { testDataObjectId, summaryVersion } = await getUnreferencedDataStoreIdAndSummaryVersion();
+
+        // Wait some time, the datastore should not be sweep ready after this wait
+        await delay(sweepTimeoutMs);
+
+        // Load a new container and summarizer based on the latest summary, summarize
+        const {
+            container: summarizingContainer2,
+            summarizer: summarizer2,
+        } = await loadSummarizerAndContainer(summaryVersion);
+
+        // Use the request pattern to get the testDataObject - this is unsafe and no one should do this in their
+        // production application - causes an inactive loaded and changed error
+        await assert.doesNotReject(
+            async () => requestFluidObject<ITestDataObject>(summarizingContainer2, testDataObjectId),
+            `Should be able to still request unreferenced datastores`,
+        );
+
+        // Send an op to update the currentTimestampMs to now
+        const mainDataStore2 = await requestFluidObject<ITestDataObject>(summarizingContainer2, "default");
+        mainDataStore2._root.set("send a", "op");
+
+        // The datastore should be tombstoned now
+        await summarize(summarizer2);
+
+        // Sending an op from a datastore substantiated from the request pattern should fail!
+        await assert.rejects(async () => requestFluidObject<ITestDataObject>(summarizingContainer2, testDataObjectId),
```

Comment:
Can we also test `handle.get`?  Seems a little weird I'll admit (since if you have the handle you probably stored it properly), but would be good to cover on account of any weird code people may write, or bugs we could have.

Reply:
Will do

Reply:
See https://dev.azure.com/fluidframework/internal/_workitems/edit/405

We can log the "viaHandle" header of the request.